templates/VALIDATION.md

```diff
@@ -22,19 +22,19 @@ validations: |
   [
     {
       "name": "validation-rule-01",
-      "valid": "/some.json.path",
-      "path”: "/some.json.path[*].leaf",
+      "valid": "$.some.json.path",
+      "path”: "$.some.json.path[*].leaf",
       "rule”: "integer",
-      "message”: "/some.json.path[*].leaf must exists",
+      "message”: "$.some.json.path[*].leaf must exists",
       "min”: 1
     },
     {
       "name": "validation-rule-02",
-      "valid": "/another.json.path",
-      "path": "/another.json.path.item",
+      "valid": "$.another.json.path",
+      "path": "$.another.json.path.item",
       "rule": "integer",
-      "message": "/another.json.path.item must be below a threshold",
-      "max": "/yet.another.json.path.defines.the.limit"
+      "message": "$./another.json.path.item must be below a threshold",
+      "max": "$.yet.another.json.path.defines.the.limit"
```

Comment:
You use `$.` here and `.` below. Care to explain the difference?

Reply:
It should be $. everywhere for consistency, even though both are legal per https://goessner.net/articles/JsonPath/index.html#e3